bin/disco_elasticache.py

```diff
@@ -50,12 +54,33 @@ def run():
             size = 'N/A'
             if cluster['Status'] == 'available':
                 size = len(cluster['NodeGroups'][0]['NodeGroupMembers'])
-            print("{0:<25} {1:5} {2:>5}".format(cluster['Description'],
-                                                cluster['Status'],
-                                                size))
+            print("{:<25} {} {} {:>5}".format(
+                cluster['Description'], cluster['ReplicationGroupId'], cluster['Status'], size))
```

Comment:
This won't be backwards compatible for any script that used to assume that the status. Not sure how much we care about that since I don't think we have any scripts that are using this